extensions/wikia/Search/classes/Test/WikiaSearchHelper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+class WikiaSearchHelperTest extends WikiaBaseTest {
+
+	public function testGetLangForSearchResultsIfResultsLangSet() {
+		$wgRequestMock = $this->getMock( 'WebRequest', [ 'getVal' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getVal" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)